pulsar-broker/src/test/java/org/apache/pulsar/schema/SchemaTest.java

```diff
@@ -205,4 +211,30 @@ public void testIsUsingAvroSchemaParser() {
             }
         }
     }
+
+    @Test
+    public void testNullKeyValueProperty() throws PulsarAdminException, PulsarClientException {
+        final String tenant = PUBLIC_TENANT;
+        final String namespace = "test-namespace-" + randomName(16);
+        final String topicName = "test";
+
+        final String topic = TopicName.get(
+                TopicDomain.persistent.value(),
+                tenant,
+                namespace,
+                topicName).toString();
+        admin.namespaces().createNamespace(
+                tenant + "/" + namespace,
+                Sets.newHashSet(CLUSTER_NAME));
+
+        final Map<String, String> map = new HashMap<>();
+        map.put("key", null);
```

Comment:
@BewareMyPower  I just expect to solve the error in this case. The `value` of `map#put(key, value)` is `null`.